lib/errors.js

```diff
@@ -29,16 +29,28 @@ class MissingPageError extends TldrError {
 Page not found.
 If you want to contribute it, feel free to send a pull request to: ${repo}
     `);
-    this.name = 'MissingPageErrror';
+    this.name = 'MissingPageError';
     // eslint-disable-next-line no-magic-numbers
     this.code = 3;
   }
 }
 
+class MissingRenderPathError extends TldrError {
+  constructor() {
+    super(trim`
+Option --render needs an argument.
+    `);
```

Comment:
Any reason not to do this, @vladimyr?
```suggestion
    super('Option --render needs an argument.');
```
Just for consistency?  It's not a problem; I'm just curious.

Reply:
Honestly, I failed to spot it. Sure let's do that 👍